Source/WebCore/platform/graphics/egl/GLContext.cpp

```diff
@@ -80,7 +80,7 @@ bool GLContext::getEGLConfig(PlatformDisplay& platformDisplay, EGLConfig* config
             WTFLogAlways("Unknown pixel layout %s, falling back to RGBA8888", environmentVariable);
     }
 
-    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
+    WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // Non-Apple ports.
```

Comment:
Seems a little Apple-centric to write a comment like this. Is there a more principled way to do this, and still achieve the goal of being able to drive these down to zero for the Apple platforms?

Reply:
Ideally I'd specify which ports are using this code. I'll try and figure this out.